flow/ondemand/client/client.go

```diff
@@ -378,6 +378,21 @@ func (o *OnDemandProbeClient) Stop() {
 	o.checkForRegistration.Stop()
 }
 
+// Invoke capture upon start of analyzer on selected from config SubGraph
```

Comment:
golink expected commend to start with name of function:

```
// InvokeCaptureFromConfig ...
```

Reply:
OK

Reply:
I would not put the provisioning code here, otherwise we will have to put provisioning stuffs (alerts, etc) everywhere in the code. I would rather create a dedicated file close to the analyzer/server.go file. So if we decide to change something we will have only one entry point to change.